modules/uhttp/http.go

```diff
@@ -84,14 +89,15 @@ type Config struct {
 type GetHandlersFunc func(service service.Host) []RouteHandler
 
 // New returns a new HTTP ModuleProvider.
-func New(hookup GetHandlersFunc, options ...ModuleOption) service.ModuleProvider {
+func New(hookup GetHandlersFunc, router *mux.Router, options ...ModuleOption) service.ModuleProvider {
```

Comment:
Sorry, I should be more specific: instead of `*mux.Router` simply accept [http.Handler](https://golang.org/pkg/net/http/#Handler), which both [gorrila.Mux](https://github.com/gorilla/mux/blob/master/mux.go#L80) and [http.ServeMux](https://golang.org/pkg/net/http/#ServeMux) implement.